api_io_sign.yaml

```diff
@@ -512,6 +522,7 @@ components:
             - WAIT_FOR_QTSP
             - SIGNED
             - REJECTED
+          $ref: "#/components/schemas/SignatureRequestStatus"
```

Comment:
isn't the status already here (previous lines)?

Reply:
I replicated what is [here](https://github.com/pagopa/io-sign/blob/02183449ae8536edecf35b3f4c430c1570e344a3/apps/io-func-sign-user/openapi.yaml#LL751C14-L751C14), I think there is an error.

Reply:
fixed!